### ISSUE TITLE
SICOMP rejected the paper

### ISSUE THREAD

**Issue by asmeurer** [sympy/sympy-paper#171]
SICOMP has rejected the paper as out of scope. [Here](https://github.com/sympy/sympy-paper/wiki/Journals) are other possible journals we can submit to. My preference is PeerJ, but I'd love to hear other's opinions as well.

**Comment by ellisonbg** [comment]
+1 on PeerJ

**Comment by scopatz** [comment]
So in order order of preference I think we should submit to these:
- ACM Transaction / SISC, only if people are interested in this
- PeerJ
- JORS

**Comment by moorepants** [comment]
+1 PeerJ

**Comment by scolobb** [comment]
The journals ACM Transation, SISC, PeerJ, and JORS seem to be appropriate targets.

We may try Applied Mathematics and Computation: they do insist on novel algorithms, but still may be interested in the SymPy paper.

**Comment by sumith1896** [comment]
We can also try SISC as it was our first preference till recently and the template is also ready.
+1 for PeerJ too.

**Comment by certik** [comment]
Yeah, why don't we try SICS next.

**Comment by fredrik-johansson** [comment]
My preference is SISC followed by TOMS.

By the way, where did you find the 20-page limit for TOMS? I can't find any information about this on the journal home page.

**Comment by asmeurer** [comment]
My feeling is that SISC would be a waste of time, since the paper is even more out of scope for it than SICOMP, based on the journal description.

**Comment by fredrik-johansson** [comment]
I don't know. I think SICOMP is more about theoretical computer science; submissions must "make a significant mathematical contribution, and be relevant to computer science". The paper doesn't really make a mathematical contribution. On the other hand, SISC has a submission category for "novel design and development of computational methods and high-quality software ... that have potentially large impact for an important class of scientific or engineering problems". I think this fits better, though the editors/referees may well be inclined to argue about the degree of novelty and whether the paper presents any new "methods".

**Comment by aterrel** [comment]
I would go with SISC then PeerJ. Last time I submitted to TOMS it took two years to get the paper published. I'm personal friends with TOMS editor so I could certainly ask what their review backlog looks like now.

**Comment by aterrel** [comment]
Also there is numerical software that does use sympy extensively, such as PetSc\* will download it and use it for FEM generation. I couldn't find a good publication but the work by Knepley has it, could add the example that might make a better case:

slides: http://www.caam.rice.edu/~mk51/presentations/PresMadisonME9642011.pdf
preprint: http://andy.terrel.us/papers_and_talks/KnepleyTerrel2012.pdf
citation: http://dl.acm.org/citation.cfm?id=2427027

AKA the microsoft of scientific computing

**Comment by fredrik-johansson** [comment]
I agree that some FEM example could help sell the paper to the average computational science type reader.

**Comment by aterrel** [comment]
Throwing another journal / magazine out there is IEEE Computing in Science and Engineering. It doesn't have the best reputation because it is half magazine. I know the editors well and am certain they would be very happy to recieve this paper. The impact factor is 1.5 over the last 5 years: https://www.researchgate.net/journal/1521-9615_Computing_in_Science_and_Engineering

**Comment by aterrel** [comment]
I guess I should clarify the reason I pointed out CISE is that the paper doesn't focus on research contributions which is usually required for research journals like SISC and SICOMP. That could be one reason it was rejected so quickly. I'm happy to email editors at SISC and TOMS to get their perspective.

**Comment by aterrel** [comment]
After reading the paper through, I have to admit it could use some work on building a coherent theme. As is I find it hard to read as something less than comprehensive technical report. By having a stronger theme that runs through the paper, it would make sense on what is in and out of the paper.

Perhaps a theme could be how ease of use enables scientific discovery...  

Additionally it doesn't seem very original, what is the goal of this paper over the original Sympy paper (which isn't actually cited)? http://wdjoyner.org/papers/oscas-sympy-cca4.pdf

**Comment by moorepants** [comment]
Andy, I agree with you. I raised this issue about the theme in #55 but have had a hard time convincing anyone of the merit. Also, few people voted on the outline in #109 to help (re)define the theme. PeerJ would be more likely to accept it as is or even something like F1000Research which explicitly states they accept "software papers". But as it stands, the paper doesn't express it's contribution to the literature in a way that would compel traditional journals to accept it. And even for journals like PeerJ and F1000Research, I think we should define a theme and then edit to reflect that decision.

**Comment by scolobb** [comment]
One subject we might consider is using SymPy in teaching.  Tools like SymPy Gamma make such applications very easy, and the fact that SymPy is an open source project gives it a definite advantage over other similar solutions.

**Comment by Upabjojr** [comment]
> One subject we might consider is using SymPy in teaching.

SymPy could be used to turn the student's focus on the subject itself, by skipping tedious calculations.

**Comment by moorepants** [comment]
There are some interesting education examples in the wild. 1) The "Doing Math with Python" book by Amit Saha uses SymPy for several of the chapters. This is targeted at high school students I believe. 2) SymPy has been used as a backend for Common Core automated testing systems in the US.

Both of these are in this zotero collection, along with my notes: https://www.zotero.org/groups/sympy/items

My guess is that there are some more cool examples that could be highlighted.

**Comment by asmeurer** [comment]
The Joyner et al. SymPy paper was written almost five years ago. A lot has changed in SymPy since then. One reason for having this paper was to coincide with the 1.0 release. This was never mentioned in the paper itself, but perhaps it should be. And for what it's worth, this paper is significantly higher quality than the Joyner at al. one.

**Comment by scolobb** [comment]
Yeah, I think we should indeed compare the new paper with the older one.  A paragraph should do.

**Comment by aterrel** [comment]
Okay SISC is out unless we want to refocus the paper on innovative concepts provided by SymPy.  I think there are some things we could tease out there such as the Riesch algorithm and innovations in numerical systems via code generation. Below is the response from the section editor who kindly reviewed the paper upon our request.

> Your manuscript provides an excellent description of the SymPy software for computer algebra. Unfortunately, the manuscript is not a good fit with the SISC editorial policy. There must be a significant innovation (algorithms, data structures, parallelization, improved efficiency on specialized hardware, etc.) that has a sizable impact on an important class of problems. In general, descriptions of software are not a good fit, although this is not a negative reflection on the software or the paper.  
> 
> Outside my official role in SISC, I’ll mention that many papers like yours are published in ACM TOMS. Another place where I have seen some papers going in PeerJ – Computer Science. A paper on the highly-regarded Clawpack software has been submitted there (https://peerj.com/preprints/1829/), for instance. (As a disclaimer, I am also on the editorial board for PeerJ.)

**Comment by asmeurer** [comment]
OK, I'd like to submit to PeerJ. Should we reformat the paper away from the SIAM templates?

**Comment by asmeurer** [comment]
See https://peerj.com/about/author-instructions/cs
